auth-server/migrations/20191120202513_refresh_token.js

```diff
@@ -5,7 +5,7 @@ exports.up = function(knex) {
 		table.string('token').unique().notNullable()
 		table.integer('user_id').notNullable()
 		table.boolean('valid').notNullable()
-		table.integer('expires').notNullable()
+		table.bigInteger('expires').notNullable()
```

Comment:
We should rather use datetime https://knexjs.org/#Schema-date

Reply:
done